flag:
  if_show_time: false

# 0 - vinilla, 1 - yolov5, 2 - euclideancluster
mode: 0

camera:
  topic: /lbas_image
  image_width: 1440
  image_height: 1080
  intrinsic:
    [
      2240.526817,
      0.000000,
      638.231713,
      0.000000,
      2232.334100,
      572.292534,
      0.000000,
      0.000000,
      1.000000,
    ]
  projection:
    [
      2199.676025,
      0.000000,
      630.104930,
      0.000000,
      0.000000,
      2211.820557,
      571.834934,
      0.000000,
      0.000000,
      0.000000,
      1.000000,
      0.000000,
    ]
  distortion: [-0.149787, 0.073690, -0.001176, -0.007231, 0.000000]

lidar:
  topic: /rslidar_points
  frame_id: rslidar
  filter:
    xmin: 0.0
    xmax: 100.0
    ymin: -100.0
    ymax: 100.0
    zmin: -100.0
    zmax: 100.0
  ec_extraction:
    cluster_tolerance: 0.1
    min_cluster_size: 20
    max_cluster_size: 25000

transform:
  lidar2camera:
    [
      0.0645287,
      -0.997914,
      0.00164512,
      -0.0251032,
      -0.0351988,
      -0.0039236,
      -0.999373,
      0.0504193,
      0.997295,
      0.0644304,
      -0.0353786,
      -0.205162,
      0.000000,
      0.000000,
      0.000000,
      1.000000,
    ]
